modules/packages/ExternalArray.chpl

```diff
@@ -25,6 +25,16 @@
 module ExternalArray {
   use ChapelStandard;
 
+  extern record chpl_external_array {
+    var elts: c_void_ptr;
+    var size: uint;
+
+    var freer: c_void_ptr;
+  }
+  extern proc chpl_make_external_array(elt_size: uint, num_elts: uint):
+    chpl_external_array;
```

Comment:
This looks odd - I'd probably give up on not overflowing the line or wrap it differently.

Reply:
Like this?
```chapel
  extern proc
  chpl_make_external_array(elt_size: uint, num_elts: uint): chpl_external_array;

  extern proc chpl_free_external_array(x: chpl_external_array);
```

Reply:
I like that better but such things are definitely subjective.